### PR TITLE
New technology partner: WordPress

### PR DESCRIPTION
**Pull request by kellyfinn**

### CODE REVIEW

**Comment by kellyfinn** [comment]
This is ready @stewsmith @julieritchie

File: website/partners/technology/wordpress/index.hbs [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    city:
+    state:
+    address1:
+    address2:
+    zip:
+    country:
+contact: 
+  techpartners@optimizely.com
+languages:
+tags:
```
Comment (stewsmith) [inline website/partners/technology/wordpress/index.hbs]: @kellyfinn please use `TR_tags`

**Comment by stewsmith** [comment]
@kellyfinn @julieritchie the logo is off center because the image has funny padding. You can right click and inspect element here: https://d1qmdf3vop2l07.cloudfront.net/optimizely-marketer-assets.cloudvent.net/raw/partner-logos/technology/wordpress.png to view it.

**Comment by kellyfinn** [comment]
@stewsmith @julieritchie Thanks Stew, logo has been updated!

**Comment by stewsmith** [comment]
@kellyfinn Logo looks good. Could you change `tags` to `TR_tags`? Thanks!

**Comment by kellyfinn** [comment]
This is ready @stewsmith

**Comment by stewsmith** [comment]
LGTM

**Pull request merged**